includes/Stories_Renderer/Renderer.php

```diff
@@ -65,6 +65,18 @@ abstract class Renderer implements RenderingInterface, Iterator {
 	 */
 	const LIGHTBOX_SCRIPT_HANDLE = 'lightbox';
 
+	/**
+	 * Object ID for the Renderer class.
+	 * To enable support for multiple carousels and lightboxes
+	 * on the same page, we needed to identify each Renderer instance.
+	 *
+	 * This variable is used to add appropriate class to the Web Stories
+	 * wrapper.
+	 *
+	 * @var int
+	 */
+	protected static $obj_id = 0;
```

Comment:
What if I use the same instance to render multiple carousels? e.g. (pseudo code):

```php
$renderer =  new Renderer();
$renderer->render();
$renderer->render();
```

This prints two carousels, and clicking on one Glider arrow button will control both carousels.

Could we just have a static `$instance_id` variable within the render method  instead  of this class property?

Reply:
Yep, you are right if the same instance is used multiple times on a page to render the stories it will have the same id. 
Didn't think of this use case or we didn't design for such case. 

Still you would want me to move this $obj_id to the method and increment if as the `->render()` is called. Will update this.

Reply:
Yeah I'd just use a static `$instance_id` variable in `render()` that we can increment.

Thanks!

Reply:
Since our `render()` method in the `Renderer::class` doesn't actually render any HTML, but the child classes override the same to render the HTML. Now if we want to have this static variable at function level then we either have to define it in the `Renderer::class->render()` and then return it so that the child classes can actually access it. or have child classes add that variable statical ( not DRY ).

How about we keep it as is and move the increment to `render()` this will keep it where it should be and not required much strings to pull. thoughts? @swissspidy

Reply:
>  How about we keep it as is and move the increment to render() this will keep it where it should be and not required much strings to pull.

Let's do this for now then  👍  Just one suggestion to use `instance_id` instead of `obj_id`, seems more descriptive.